.agents/tools/marketing/ad-creative/campaign-management.md

```
@@ -0,0 +1,2913 @@
+
+---
+
+1. OBJECTIVE
+What is the goal of this creative?
+
+Example: "Drive free trial signups for our new project management software"
+
+---
+
+2. TARGET AUDIENCE
+Who is this for?
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)